corehq/apps/es/utils.py

```diff
@@ -1,8 +1,35 @@
+import json
 from datetime import timezone, datetime
+
 from django.conf import settings
+
+from corehq.util.es.elasticsearch import SerializationError
+from corehq.util.json import CommCareJSONEncoder
 from corehq.util.metrics import metrics_counter
 
 
+class ElasticJSONSerializer(object):
```

Comment:
Yes.

Disclosure: throughout this PR there has been a (perhaps inconsistent) effort to try avoid using the `ES` acronym since it can cause ambiguities with ECMAScript (this will become more problematic as we move towards Elastic 6 and Elastic 7).

When moving this from the `corehq.elastic` module, I took the opportunity to rename it for that reason.  I'm not embedded on the decision to rename it and I could go both ways.

Further disclosure: I really wish this app was named `corehq.apps.elastic`, but the idea of renaming it broke my invasivity meter.